lib/workers/repository/index.ts

```diff
@@ -138,3 +140,23 @@ function emptyExtract(config: RenovateConfig): ExtractResult {
     packageFiles: {},
   };
 }
+
+export function printRepositoryProblems(config: RenovateConfig): void {
+  const repoProblems = new Set(
+    getProblems()
+      .filter(
+        (problem) =>
+          problem.repository === config.repository && !problem.artifactErrors
+      )
+      .map(
+        (problem) =>
+          `${nameFromLevel[problem.level].toUpperCase()}: ${problem.msg}`
+      )
+  );
```

Comment:
Please make this shared function between here and dashboard so that we don't have it duplicated